### ISSUE TITLE
Option to add all groups to node of choosing on `verdi import`

### ISSUE THREAD

**Issue by sphuber** [aiidateam/aiida_core#2075]
I have the use case where I have an export archive with a large number of data nodes, `StructureData` in this case. When importing I would like to be able to automatically add them to a `Group` that may not already exist. Or alternatively, if the nodes are already in a group in the export file, I might want to change the name.

**Comment by giovannipizzi** [comment]
Every time an import is done, nodes are added to an auto group.

Tasks:
- [ ] add documentation about auto groups and about the fact that nodes are automatically added to a group upon import.
- [ ] Have an option in `verdi import` to create a "normal" group with a given name instead of the auto group
- [ ] Have utilities to easily rename groups from verdi (so one can put things into a group after importing, if he realises that he wants to do it too late). This is also useful for other purposes. This should not only allow to change the group name but also the type (at least from a custom one to a user-specified one). Or better, to avoid to have to think to all edge cases, maybe a rename should just change the name, and then we should have a 'copy group' that allows to create a new group also with a different type.

**Comment by yakutovicha** [comment]
The possibility to rename groups is already present:
```
verdi group rename --help
Usage: verdi group rename [OPTIONS] GROUP NAME

  Rename an existing group. Pass the GROUP for which you want to rename and
  its new NAME.

Options:
  -h, --help  Show this message and exit.
```

However, it is not possible to change the group type. Will add this possibility as well

**Comment by sphuber** [comment]
Fixed in #2329
